content/02.body.md

```diff
@@ -207,6 +207,45 @@ We have considered the edge prior as a baseline edge predictor, whose performanc
+First, the performance of the edge prior was compared to determine the performance attributable to the degree sequence of the PPI network.
+The first comparison gave insight into the ability of the PPI network to be reconstructed by degree.
+Second, the five edge prediction features were computed on 100 permuted networks and used to reconstruct the unpermuted network.
+Each permuted network corresponded to AUROC values quantifying the performances of features computed on it.
+The second comparison gave insight into the performance of each feature if the feature was only picking up on degree.
+
+![Network reconstruction performances by five edge prediction features.
+  Dotted red line indicates performance of the edge prior.
+  Each feature was computed on the unpermuted and 100 permutations of the STRING PPI network.
+  ](https://github.com/greenelab/xswap-analysis/raw/4d9137f43a16fbe6c6a02865adceaa9a3195fe2d/img/feature-auroc.png){#fig:feature-auroc width="85%"}
+
+The edge prior was able to reconstruct the PPI network with an AUROC of 0.797 (dotted red line in Figure {@fig:feature-auroc}).
+Since the edge prior captures the performance of degree, this result indicated that nonspecific predictions alone showed this level of reconstruction performance.
+In the second comparison, edge prediction features computed on permuted networks showed performance equal or lower to their performances on the unpermuted networks.
+This indicated that four out of five edge prediction features picked up on more than node degree for the prediction task.
+The preferential attachment index is the product of source and target degree, and we found that its performance did not differ from the edge prior or the feature's performance when computed on permuted networks.
+The four remaining features showed far higher reconstruction performance than the edge prior or feature values computed on permuted networks.
+
+The comparison of features demonstrated that some features (eg: RWR) pick up on far more than degree for the prediction task, while others (preferential attachment index) do not.
+RWR picked up on more than degree because it reached all of a node's neighbors in one step after a restart.
+Naturally, the preferential attachment index did not pick up on more than degree because its value is simply the product of source and target degrees.
+The edge prior provided the baseline level of performance that is attributable to degree alone.
+Features whose performances on permuted networks were below that of the edge prior only imperfectly pick up on degree (eg: Jaccard index), whereas those with performances exceeding the edge prior pick up on more than degree.
+The no-holdout reconstruction task we conducted was intentionally simple to demonstrate that the edge prior and permutation framework captures the effects of degree in a task where the results could be intuited.
+
```

Comment:
I struggled a bit with the paragraph above. Can you see if it has a topic sentence, details proceeding in order in support of that topic, and then a concluding sentence? I think it has the first two, but in the middle it feels like it's jumping around a little.

Also, I wasn't a fan of "pick up on" as it is probably too informal. Is there something else you can use instead? I tried "captured" but I'm not sure that quite gets it either.